CONTRIBUTING.md

```
@@ -326,4 +316,4 @@ Before releasing a new version:
 ## Questions?
 
 Refer to the main [README.md](README.md) for general information.
-Open an issue if you have questions or suggestions!
+Open an issue if you have questions or suggestions!
```

markdownlint-cli2 (0.20.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)